crypto/keygen.py

```diff
@@ -0,0 +1,48 @@
+from Crypto.PublicKey import RSA
+from Crypto.Signature.pkcs1_15 import PKCS115_SigScheme
+
+
+def generate_key_pair(node_id):
+
+    key = RSA.generate(2048)
+    file_priv = "crypto/" + str(node_id) + "-private.pem"  # Find better way to do these names
```

Comment:
This is great to start. We might think about going a little further and add a keystore manager like this one: https://pypi.org/project/pyjks/